static/assets/scripts/s.js

```diff
@@ -284,7 +284,7 @@ function newCustomTheme() {
     alert("Name cannot contain a comma.")
     return
   }
-  if (!localStorage.getItem("customThemes")) {
+  if (localStorage.getItem("customThemes" !== "")) {
```

Comment:
kek you could've just removed the ! from before

Reply:
im so confused rn

Reply:
` if (localStorage.getItem("customThemes")) {`

do that

Reply:
I got it